### PR TITLE
rectify file stats test case

### PR DESCRIPTION
**Pull request by littlestar642**
Overview

One of the tests was written incorrectly. 
I have rectified it and it works fine now.

### CODE REVIEW

**Comment by stewartie4** [comment]
@littlestar642 how does this relate to
https://github.com/0chain/system_test/pull/100

**Comment by Kishan-Dhakan** [comment]
I renamed the PR to refer to `file stats` correction instead of `file update`

File: tests/cli_tests/zboxcli_file_stats_test.go [hunk 1]
```diff
@@ -513,15 +501,13 @@ func TestFileStats(t *testing.T) {
 			require.Equal(t, fmt.Sprintf("%x", sha3.Sum256([]byte(allocationID+":"+remoteFilePath))), data.PathHash)
 			require.Equal(t, int64(0), data.NumOfBlockDownloads)
 			require.Equal(t, float64(data.NumOfBlocks), math.Ceil(float64(data.Size)/float64(chunksize)))
+			require.Equal(t, int64(2), data.NumOfUpdates, "the number of updates count should increment")
 			if data.WriteMarkerTxn == "" {
 				require.Equal(t, false, data.BlockchainAware)
 			} else {
 				require.Equal(t, true, data.BlockchainAware)
 			}
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_file_stats_test.go:509]: Since file size is updated, assert on data.size too?

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: tests/cli_tests/zboxcli_file_stats_test.go [hunk 2]
```diff
@@ -513,15 +502,14 @@ func TestFileStats(t *testing.T) {
...
 
-			//FIXME: POSSIBLE BUG: The update did not get reflected in the file stats
-			require.NotEqual(t, int64(2), data.NumOfUpdates, "the number of updates count should increment")
-
 			//FIXME: POSSIBLE BUG: key name and blobberID in value should be same but this is not consistent for every run and happening randomly
 			// require.Equal(t, blobberID, data.BlobberID, "key name and blobberID in value should be same")
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_file_stats_test.go:514]: @littlestar642 should this still be commented out?

**Pull request merged**